style.css

```
@@ -1,466 +1,263 @@
-  left: 0;
-}
 
-/* Этот класс отвечает за заблюренный фон. */
-.mobile__menu__background {
-  position: fixed;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  top: 0;
-  z-index: 99;
+	
 
-  opacity: 0;
-  visibility: hidden;
 
-  backdrop-filter: saturate(180%) blur(10px);
-  background-color: rgba(50, 48, 48, 0.55);
-  transition: all 0.5s ease;
+.services-item__title {
+    font-size: 14px;
+    line-height: 18px;
+    margin-bottom: 6px;
 }
 
-/* https://www.w3schools.com/cssref/sel_element_pluss.php#:~:text=that%20are%20placed-,immediately%20after,-%3Cdiv%3E%20elements%3A */
-.mobile__menu.active + .mobile__menu__background {
-  opacity: 1;
-  visibility: visible;
-  z-index: 99;
+.image__img__item{
+	position: absolute;
+	margin-bottom: -52px;
 }
+.hservices__list {
+	display: flex;
+    justify-content: center;
+	 margin: 0 -16px 88px;
 
-.mobile__menu__close__button {
-  all: unset;
-  position: absolute;
-  top: 12px;
-  right: 12px;
-}
-.mobile__menu__list{
- margin-top: 50px;
- margin-left: 16px;
-}
-.mobile__menu__list__item{
-  list-style: none;
-  font-size: inherit;
-  color: inherit
-}
-.mobile__menu__list__item a{
-  text-decoration: none;
-  color: #231f20;
-}
-.mobile__menu__contact{
-padding: 0 16px;
-}
-.mobile__menu__contact__span{
-  font-style: normal;
-  font-weight: 300;
-  font-size: 12px;
-  line-height: 100%;
-  color: var(--c-gray);
 }
-.mobile__menu__contact__item a{
-  text-decoration: none;
-  color: #231f20;
-  font-weight: 500;
-}
-.mobile__menu__contact__icon{
-  display: flex;
-  align-items: center;
-  flex-direction: row;
-  text-decoration: none;
-  color: var(--black);
-  font-weight: var(--weight-500);
-  text-decoration: none;
-  gap: 3px;
-}
-.mobile__menu__socialc__adress{
-  display: flex;
-  grid-gap: 5px;
-  position: relative;
-  margin-top: 8px;
-  font-style: normal;
-  font-weight: 300;
-  font-size: 14px;
-  line-height: 17px;
-  color: var(--c-dark);
-}
-
-.mobile__menu__socialc__email{
-  display: flex;
-  gap: 5px;
-
-  position: relative;
-  margin-top: 8px;
-  font-style: normal;
-  font-weight: 300;
-  font-size: 14px;
-  line-height: 17px;
-  color: var(--c-dark);
- text-decoration: none;
+.item__content__title{
+	margin-bottom: 8px;
+    display: block;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
 }
-.mobile__menu__socialc__email a{
-  font-style: normal;
-  font-weight: 300;
-  font-size: 14px;
-  line-height: 17px;
-  color: var(--c-dark);
- text-decoration: none;
+.item__content__title:hover{
+	color: var(--gray);
 }
+.item__content__summary{
+	margin-bottom: 10px;
+    display: block;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 15px;
+    line-height: 140%;
+    color: var(--gray);
+}
+.item__content__price{
+	display: flex;
+	font-style: normal;
+	font-weight: 500;
+	font-size: 26px;
+	line-height: 36px;
+	color: var(--orange);
+}
+.image{
+	height: 192px;

```

height удаляем, и даем внешний минусовой отступ вниз чтобы потянуть спан с картинкой вниз на блок с текстом. Чтобы стап был выше слоя блока с текстом ему так же нужно дать z-index: 1;

```suggestion
    margin-bottom: -40px;
    z-index: 1;
```